lib/ReactGridLayout.jsx

```diff
@@ -96,6 +96,9 @@ const compactType = (props: Props): CompactType => {
   return verticalCompact === false ? null : compactType;
 };
 
+const layoutClassName = "react-grid-layout";
+const isFirefox = navigator.userAgent.toLowerCase().includes("firefox");
```

Comment:
We're doing this check anyway, can we perhaps take advantage of it and run `e.dataTransfer.setData('text/plain', '')` for the user within `onDragStart` perhaps?

Reply:
@n1ghtmare we can't do that because users should place it into their own code (in an element outside of ReactGridLayout)

Reply:
It's not about our implementation, it's about "draggable" implementation in firefox